### PR TITLE
Add social media links to the about screen.

### PR DESCRIPTION
**Pull request by hblumberg**
Addresses: https://github.com/5calls/android/issues/60

Test Plan:
- Open the about screen.
- See social media links; click each and see intent launcher with the appropriate prompt. Ensure selecting each opens the correct url.

screenshot_1505605413

### CODE REVIEW

**Comment by hblumberg** [comment]
screenshot_1505608717

(I couldn't find a good way to do this in XML, but happy to switch if you know of ways!)

File: 5calls/app/src/main/java/org/a5calls/android/a5calls/controller/AboutActivity.java [hunk 1]
```diff
@@ -39,12 +41,20 @@
     private static final String TAG = "AboutActivity";
     private static final String EMAIL_CONTENT_TYPE = "message/rfc822";
 
+    // Links to social media accounts.
+    private static final String TWITTER_URL = "https://twitter.com/make5calls";
```
Comment (dektar) [inline 5calls/app/src/main/java/org/a5calls/android/a5calls/controller/AboutActivity.java]: Could you please keep these in strings.xml with translatable=false instead?

File: 5calls/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -70,6 +70,18 @@
     <!-- Button that allows users to send 5 Calls an email [CHAR_LIMIT=30] -->
     <string name="contact_us_btn">contact us</string>
 
+    <!-- Prompt to connect with 5 Calls through social media [CHAR_LIMIT=100] -->
+    <string name="social_prompt">You can also get in touch via</string>
+
+    <!-- Button that links to 5 Calls' Twitter -->
```
Comment (dektar) [inline 5calls/app/src/main/res/values/strings.xml]: add a CHAR_LIMIT for consistency.

File: 5calls/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -70,6 +70,18 @@
...
+    <string name="twitter_btn">Twitter</string>
```
Comment (dektar) [inline 5calls/app/src/main/res/values/strings.xml]: Should these 3 buttons be translatable? Are the words "Facebook"/"Twitter"/"Instagram" the same in all languages? If not, put translatable=false in the string tag.

File: 5calls/app/src/main/res/layout/activity_about.xml [hunk 4]
```diff
@@ -126,6 +126,69 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="0dp"
+                android:gravity="center"
+                android:background="@android:color/transparent"
+                android:minWidth="@dimen/accessibility_min_size"
+                android:minHeight="@dimen/accessibility_min_size"
+                android:text="@string/instagram_btn"
+                android:textColor="@color/colorAccent"
+                android:textAppearance="@style/TextAppearance.AppCompat.Widget.Button"
```
Comment (dektar) [inline 5calls/app/src/main/res/layout/activity_about.xml]: not sure you need this textAppearance line here and in the other buttons.

**Comment by chadschultz** [comment]
@hblumberg I see this PR is several years old and has not been completed. Are you interested in finishing it up? If not, at some point in the next few weeks (no promises) I may make your changes in a separate branch and ask an admin to close this out.

**Comment by chadschultz** [comment]
Created PR #121 as a replacement for this PR.

**Pull request closed without merging**